src/Bones/babylon.bone.ts

```diff
@@ -551,26 +612,41 @@
 
         }
 
-        public getRotation(mesh: AbstractMesh): Quaternion {
+        public getRotation(space = Space.LOCAL, mesh?: AbstractMesh): Quaternion {
 
             var result = Quaternion.Identity();
 
-            this.getRotationToRef(mesh, result);
+            this.getRotationToRef(result, space, mesh);
 
             return result;
 
         }
 
-        public getRotationToRef(mesh: AbstractMesh, result: Quaternion): void{
+        public getRotationToRef( result: Quaternion, space = Space.LOCAL, mesh?: AbstractMesh): void{
+

```

Comment:
same here. I'm not convinced we should change the order. 
I would prefer using overloads instead:
public getRotationToRef(result: Quaternion):
public getRotationToRef(space ,result: Quaternion):
public getRotationToRef(space = Space.LOCAL, mesh: AbstractMesh = null, result: Quaternion):